test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -1380,28 +1380,78 @@ public async Task NoResetOnClose(bool noResetOnClose)
                     : original));
         }
 
-        #region pgpass
-
         [Test]
         [NonParallelizable]
         public async Task UsePgPassFile()
         {
-            var builder = new NpgsqlConnectionStringBuilder(ConnectionString)
-            {
-                IntegratedSecurity = false
-            };
+            using var resetPassword = SetEnvironmentVariable("PGPASSWORD", null);
```

Comment:
nit: can do `using var _` if you don't access `resetPassword` below (same in the other test)

Reply:
Unfortunately you can't do this twice, so I do `using var __` and so on in some tests :/

Reply:
That's why I haven't used them here. Underscores are bad for naming.

Reply:
I agree it's problematic... with `resetPassword` the IDE gives an "unused variable" warning, which also sucks. No perfect solution..